### PR TITLE
print (WIP), fake scan (WIP) and text conversion for ultra-lite

### PR DESCRIPTION
**Pull request by Frooodle**
# Description

Please provide a summary of the changes, including relevant motivation and context.

Closes #(issue_number)

## Checklist:

- [ ] I have read the [Contribution Guidelines](https://github.com/Stirling-Tools/Stirling-PDF/blob/main/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings

## Contributor License Agreement

By submitting this pull request, I acknowledge and agree that my contributions will be included in Stirling-PDF and that they can be relicensed in the future under the MPL 2.0 (Mozilla Public License Version 2.0) license.

(This does not change the general open-source nature of Stirling-PDF, simply moving from one license to another license)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,15 +25,17 @@ ENV DOCKER_ENABLE_SECURITY=false \
 RUN echo "@testing https://dl-cdn.alpinelinux.org/alpine/edge/main" | tee -a /etc/apk/repositories && \
     echo "@testing https://dl-cdn.alpinelinux.org/alpine/edge/community" | tee -a /etc/apk/repositories && \
     echo "@testing https://dl-cdn.alpinelinux.org/alpine/edge/testing" | tee -a /etc/apk/repositories && \
+    apk update && \
```
Comment (Zoey2936) [inline Dockerfile:28]: why are downloading the cache, never use it and then delete it at the end again?

Comment (Frooodle) [reply Dockerfile:28]: Good point, will remove this on next branch

Comment (Zoey2936) [reply Dockerfile:28]: just want to remind you that it is still no changed

Comment (Zoey2936) [reply Dockerfile:28]: just want to remind you that it is still no changed

**Pull request merged**